python/LIController.py

```diff
@@ -215,12 +222,25 @@ def GetFiducialVolume(self):
         """
         :return: identified fiducial volume for the experiment, None if not found
         """
-        fid_vol = None
-        for sector in self.detector_model.Sectors:
-            if self.experiment in fid_vol_dict.keys():
-                if sector.name == fid_vol_dict[self.experiment]:
-                    fid_vol = sector.geo
-        return fid_vol
+        if self.experiment=="MiniBooNE":
+            return _geometry.Sphere(_geometry.Placement(_math.Vector3D([0,0,0])),
+                                    5.0, 0)
+        elif self.experiment=="CCM":
+            return _geometry.Cylinder(_geometry.Placement(_math.Vector3D([23,0,-0.65])),
+                                      0.96, 0, 1.232)
+        elif self.experiment=="MINERvA":
+            edges = [[0.000, 0.93675],
+                    [0.81125, 0.46838],
+                    [0.81125, -0.46838],
+                    [0.000, -0.93675],
+                    [-0.81125, -0.46838],
+                    [-0.81125, 0.46838]]
+            zsecs = [_geometry.ZSection(1.45,0,1),
+                     _geometry.ZSection(4.0,0,1)]
+            return _geometry.ExtrPoly(_geometry.Placement(_math.Vector3D([0,0,0])),
+                                      edges,zsecs)
+        else:
+            return None
```

Comment:
I know we discussed where we wanted to put the fiducial volume information before, and decided it should probably be something user defined / in the code. But, if it's going to have some defaults that get used automatically via LIController, then it probably does make more sense to put them in the resources directory somewhere and version them. It would also make sense to provide the user with a mechanism to override these settings easily.

Maybe we could put them into the geometry .dat files as something that the DetectorModel does not parse, but that is retrieved by LIController? Or it could be in a separate folder since we already have folders for densitites and materials?

Reply:
This has been addressed in the one of the most recent commits—fiducial volumes are now defined in the geo .dat files